aiida/storage/sqlite_temp/backend.py

```diff
@@ -51,7 +51,8 @@ class Configuration(BaseModel):
             default_factory=mkdtemp
         )
 
-    _read_only = False
+    read_only = False
```

Comment:
Can't we remove this one, since `read_only` is set to `False` by default on the parent class?

Reply:
Yes, this shouldn't be here, will remove it